test/validators.js

```diff
@@ -2992,14 +2992,16 @@ describe('Validators', () => {
       validator: 'isPassportNumber',
       args: ['RU'],
       valid: [
-        '26 32 636829',
```

Comment:
Are the 10-digit ones invalid or you only intended to also support 9-digits?

Reply:
@profnandaa check #1807 for more context, actually the 10 digit passport number is the Russian equivalent of Identity Card and shouldn't be included in the `isPassportNumber` validator

Reply:
@tux-tn -- I see, thanks for the clarification!